introduction/principles.md

```diff
@@ -19,16 +19,17 @@ Let's take a look at each of these principles in more detail.
 ## Openness
 
 The configuration of an open project enables frictionless contributing.
-Projects should be discoverable and well documented through README.md and CONTRIBUTING.md files within the root of the repo.
+Projects should be discoverable and well-documented through README.md and CONTRIBUTING.md files within the root of the repo.
 Anyone within an organization should be able to find a desired project and ramp up without an inordinate amount of direct guidance from host team members.
 Host team contact information should be prevalent with as many channels as makes sense for the project.
-Host team intentions to InnerSource their project should be shared through relevant organization channels to raise awareness. Particularly in smaller 
-settings you may want to establish a regular "broadcast" on the InnerSource work your team is doing, in larger settings this may create a lot of 
-noise and making sure it is discoverable in a easy to use tool is probably a better path. Remember, the goal is awareness use the appropriate channels that 
+Host team intentions to accept InnerSource contributions to their project should be shared through relevant organization channels to raise awareness. Particularly in smaller 
+settings you may want to establish a regular "broadcast" on the InnerSource work your team is doing.
+In larger settings, however, such broadcast may create a lot of 
+noise, and it may be more approproate to make sure the project is discoverable in a easy-to-use tool. Remember, the goal is awareness use the appropriate channels that 
```

Comment:
>     * "...discoverable in a easy-to-use tool." should be AN "...discoverable in an easy-to-use tool."

Could you make that an edit suggestion? Then we can just one-click commit it and have your contribution recorded with name too. (To enable making a co-author list)

> 
>     * Also...i'm having an issue with easy-to-use tool
>       
>       * What are we recommending here...what is a hard to use tool for discovery?
>       * Should this be a home grown tool that everyone can use (we have one, not perfect, to me it's easy but does everyone else find it to be easy)

From my point-of-view: We're not recommending anything, mostly because there is no one true tool for this. There's a pattern on InnerSource portals from what I remember. 
Depending on your org's size and structure the right tool can differ. Maybe this topic would be a case for #62.

Reply:
Could we try to bring the line breaks to the "one sentence, one line" and "no artificial line breaks, e.g. after 80 characters" writing scheme listed in the [CONTRIBUTING.md](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/CONTRIBUTING.md)

Reply:
Adjusted line breaks.